rafx-api/Cargo.toml

```diff
@@ -43,11 +43,21 @@ dispatch = { version = "0.2", optional = true }
 foreign-types-shared = { version = "0.1", optional = true }
 block = { version = "0.1.6", optional = true }
 
+# gl
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+raw-gl-context = { git = "https://github.com/aclysma/raw-gl-context.git", branch = "rafx", optional = true }
```

Comment:
This is setting off the `cargo-deny` on the PR, I think.

Reply:
I've been in contact with the author of that crate. There are a few changes that are necessary. Hopefully they can publish something in the next week or two, but otherwise I'll need to embed the crate's code within rafx. I don't want to block committing this on it.

Reply:
Would that break all subsequent PR jobs until either 1.) the crate is fixed, or 2.) you embed that crate?

Reply:
I might go ahead and embed the crate, can always pull it out if the upstream crate publishes. It's not a big project, the main downside is that it add some noise to the cargo.toml.

Reply:
I think I'd prefer embedding for now until we have a different solution, if only to unbrick the CI jobs. 🤷